orte/tools/orterun/help-orterun.txt

```diff
@@ -660,3 +660,8 @@ method and try launching your job again.
 
 Your job will now abort.
 #
+[no_debugger_launch_support]
+This release of Open MPI does not support launching of MPI jobs with
+a debugger using mpirun.  A debugger can be attached to an MPI job
+previously launched using mpirun.
```

Comment:
This statement is not quite true -- it works fine with DDT.  I think you're trying to say that the "--debug" option is not supported in this release, right?  Perhaps:

> This version of Open MPI is known to have a problem using the "--debug" option to mpirun, and has therefore disabled it.  This functionality will be restored in a future version of Open MPI.
>
> Please see https://github.com/open-mpi/ompi/issues/1225 for further details.

(note that Github abbreviated the full URL for open-mpi/ompi#1225 in the rendering of this comment -- I'd put the full https URL in the help message).